enterprise/backend/test/metabase_enterprise/caching/strategies_test.clj

```diff
@@ -28,29 +28,29 @@
           (is (= {:type     :duration
                   :duration 1337
                   :unit     "hours"}
-                 (:cache-strategy (#'qp.card/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
+                 (:cache-strategy (#'qp.card/query-for-card card [] {} {} {:dashboard-id (u/the-id dash)}))))))
```

Comment:
Was this just a typo before, or does this function support both sequences and maps? If so, what's the difference in their meaning?

Reply:
Sooo it was an honest error which had some complaints in the REPL which I did not notice before - since they started not too long ago. I.e. when I wrote this code it did not complain about wrong schema yet, and I've just copied this code from somewhere else.